<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoDialog</class>
 <widget class="QDialog" name="DemoDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Рандом</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QListWidget" name="MainFrame">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>72</pointsize>
             </font>
            </property>
            <property name="cursor" stdset="0">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="autoScroll">
             <bool>true</bool>
            </property>
            <property name="autoScrollMargin">
             <number>16</number>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="flow">
             <enum>QListView::TopToBottom</enum>
            </property>
            <property name="resizeMode">
             <enum>QListView::Fixed</enum>
            </property>
           </widget>
           <widget class="QListWidget" name="ClassList"/>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QCheckBox" name="chooseAllCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="classNameLabel">
           <property name="text">
            <string>Загрузите класс...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="EditClassButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string> Редактировать класс </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="textButton">
           <property name="text">
            <string> Создать класс </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>99</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>72</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonFirstGroup">
             <property name="text">
              <string>1 группа</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSecondGroup">
             <property name="text">
              <string>2 группа</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="StartFromTheBegining">
               <property name="text">
                <string>Заново</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="BringToTheBoardButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string> Вызвать следующего </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BringToTheBoardButton</sender>
   <signal>clicked()</signal>
   <receiver>MainFrame</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>691</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
